speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/racescreens/racestopmenu.cpp

```diff
@@ -406,11 +412,12 @@ RmStopRaceMenu()
     // Attempt to find a human driver
     for (j=0; ; j++) {
         snprintf(buf, sizeof(buf), "%s/%s/%d", ROB_SECT_ROBOTS, ROB_LIST_INDEX, j+1);
-        test_name = GfParmGetStr(hdHandle, buf, ROB_ATTR_NAME, "");
+        human_test_name = GfParmGetStr(hdHandle, buf, ROB_ATTR_NAME, "");
```

Comment:
zou hier ook nog ASSISTED HUMAN DRIVER: moeten staan?

Reply:
Ik heb het boven de functie gezet omdat in de functie dingen aangepast worden en het veel wordt om het bij elke regel te zetten